test/ros_parameter_handler_test.cpp

```diff
@@ -53,22 +55,23 @@ class ROSParameterHandlerTest : public ::testing::Test
       host_udp_port_      =        12345;
 
       // Default expected values
-      default_password_       =   "admin";
-      default_frame_id_       = "scanner";
-      default_skip_           =         0;
-      default_publish_topic_  =    "scan";
-      default_angle_start_    =         0;
-      default_angle_end_      =      2750;
-
+      default_password_        =   "admin";
+      default_frame_id_        = "scanner";
+      default_skip_            =         0;
+      default_publish_topic_   =    "scan";
+      default_angle_start_     =         0;
+      default_angle_end_       =      2750;
+      default_x_axis_rotation_ =   275./2.;
```

Comment:
I'm not sure if adding this calculation makes the default value any more apparent. I think we should just solve that by an include. Of course that would require all parameters to have a default value defined in some variable.